data/tools/wmlunits

```diff
@@ -66,6 +66,52 @@ def get_info(addon):
         print(e)
     return _info[addon]
 
+def get_list_batch_paths(list, batch, output):
+    # Excluding the len() > 1 cases,
+    # there are 3x3 branches to handle.
+    # Bij: list = [None, [], [FILE]][i]
+    # Bij: batch = [None, [], [FILE]][j]
+
+    default_yml_file = os.path.join(output, 'list.yml')
+    if list is None and batch is None:
```

Comment:
This long `if` block looks really confusing. I'd like it to first check for the `list` value, then under every one of the three possible values check the value of `batch`. Also, `list()` is a built-in function, so you may want to replace it with `list_` or some other variable name.